src/planner/vectorfield/VectorFieldPlanner.cpp

```diff
@@ -196,29 +198,24 @@ std::unique_ptr<aikido::trajectory::Spline> planPathByVectorField(
            && terminationStatus
                   != VectorFieldPlannerStatus::CACHE_AND_TERMINATE);
 
-  if (cacheIndex >= 0)
+  // Print the termination condition.
```

Comment:
Is this refactor actually correct? This was a somewhat hasty change to stop throwing exceptions.

Reply:
Yes. I think the logic is the same with previous one. The difference is that no exception will be thrown as long as a planned trajectory is returned.